### PR TITLE
fix issue:GetReminder throws exception when reminder don't exists #1167

### PR DESCRIPTION
**Pull request by EranOfer**

### CODE REVIEW

File: src/TestGrains/ReiminderTestGrain.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using Orleans;
+using System.Threading.Tasks;
+using UnitTests.GrainInterfaces;
+
+namespace UnitTests.Grains
+{
+    internal class ReiminderTestGrain : Grain, IReminderTestGrain
```
Comment (veikkoeeva) [inline src/TestGrains/ReiminderTestGrain.cs]: "ReiminderTestGrain", both in the class name and file name. :)

File: src/TestGrainInterfaces/IReminderTestGrain.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using Orleans;
+using System.Threading.Tasks;
+
+namespace UnitTests.GrainInterfaces
+{
+    public interface IReminderTestGrain : IGrainWithIntegerKey
+    {
+        Task<bool> IsReiminderExists(string reiminderName);
```
Comment (veikkoeeva) [inline src/TestGrainInterfaces/IReminderTestGrain.cs]: "IsReiminderExists", I suppose this should be IsReminderExists".

File: src/Tester/ReminderTest.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        {
+        }
+
+        [ClassCleanup]
+        public static void MyClassCleanup()
+        {
+            StopAllSilos();
+        }
+
+        [TestMethod]
```
Comment (gabikliot) [inline src/Tester/ReminderTest.cs]: Please add [TestMethod, TestCategory("BVT"), TestCategory("Functional"), TestCategory("Reminders")]

**Comment by gabikliot** [comment]
All looks good now. You just need to rebase, due to a conflict probably in the Tester project.

**Comment by EranOfer** [comment]
Rebased to master.

**Comment by gabikliot** [comment]
Thank you @EranOfer !

**Pull request merged**